### PR TITLE
Fixes #6594: Do not run update-rc.d when it is not needed

### PR DESCRIPTION
**Pull request by Kegeruneku**
Ticket: http://www.rudder-project.org/redmine/issues/6594

To be merged in 3.1

### CODE REVIEW

File: rudder-inventory-ldap/debian/rules [hunk 1]
```diff
@@ -84,7 +84,7 @@ binary-arch: install
 #	dh_installlogrotate
 #	dh_installmime
 #	dh_python
-	dh_installinit --update-rcd-params=disable -R --name=rudder-slapd # restart after upgrade (instead of stop, upgrade, start)
+#	dh_installinit --no-start # restart after upgrade (instead of stop, upgrade, start)
```
Comment (peckpeck) [inline rudder-inventory-ldap/debian/rules]: The comment doesn't match the code

File: rudder-jetty/debian/rules [hunk 2]
```diff
@@ -56,7 +56,7 @@ binary-arch: build install
 #	dh_installpam
 #	dh_installmime
 #	dh_python
-	dh_installinit --update-rcd-params=disable --no-start
+#	dh_installinit --no-start # restart after upgrade (instead of stop, upgrade, start)
```
Comment (peckpeck) [inline rudder-jetty/debian/rules]: odem

File: rudder-inventory-ldap/debian/rules (hunk 1 above)
Comment (peckpeck) [inline rudder-inventory-ldap/debian/rules]: Does this work when there is no service at all ?

Comment (Kegeruneku) [inline rudder-inventory-ldap/debian/rules]: No, this is why it is commented :)

**Comment by peckpeck** [comment]
OK Good

**Pull request merged**